src/runtime_src/core/pcie/driver/linux/xocl/subdev/qdma.c

```diff
@@ -1250,7 +1250,7 @@ static ssize_t queue_write_iter(struct kiocb *kiocb, struct iov_iter *io)
 	qdma = queue->qdma;
 
 	nr = io->nr_segs;
-	if (!iter_is_iovec(io) || nr != 2) {
+	if (!iter_is_iovec(io) || (nr & 0x1)) {
```

Comment:
what is this change for?

Reply:
Hi, Lizhi. This is to allow multiple requests in 1 io_submit(), each request will take 2 buffers.